qa/tasks/cephfs/test_dump_export_states.py

```diff
@@ -0,0 +1,95 @@
+from teuthology.contextutil import safe_while, MaxWhileTries
+from tasks.cephfs.cephfs_test_case import CephFSTestCase
+import logging
+
+log = logging.getLogger(__name__)
+
+class TestDumpExportStates(CephFSTestCase):
+    MDSS_REQUIRED = 2
+    CLIENTS_REQUIRED = 1
+
+    EXPORT_STATES = ['locking', 'discovering', 'freezing', 'prepping', 'warning', 'exporting']
+
+    def setUp(self):
+        CephFSTestCase.setUp(self)
```

Comment:
you can change this to `super().setUp()`

Reply:
> you can change this to `super().setUp()`

OK, I have modified it.